cmd/kogito/command/converter/binarybuildtype_converter_test.go

```diff
@@ -0,0 +1,30 @@
+// Copyright 2020 Red Hat, Inc. and/or its affiliates
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package converter
+
+import (
+	"github.com/kiegroup/kogito-cloud-operator/cmd/kogito/command/flag"
+	"github.com/kiegroup/kogito-cloud-operator/cmd/kogito/command/util"
+	"github.com/kiegroup/kogito-cloud-operator/pkg/apis/app/v1alpha1"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func Test_FromArgsToBinaryBuildType(t *testing.T) {
```

Comment:
Don't need to change, but in the future you could consider table-driven tests: https://github.com/golang/go/wiki/TableDrivenTests

Reply:
I changed it, but I'm not sure if I did it properly.

Reply:
Better! Which IDE do you use? VSCode and Goland have a feature that you can generate an unit test for a given function. The base template is a table driven one. You can start from there.

Reply:
I use Vim, but I found a [plugin](https://github.com/cweill/gotests) that has the same functionality. I updated the tests to use this templating.